File: scripts/number_range.js

function numberRange(number)
{
  if (number >= 50 && number <= 200)
  {
    return "Number is in range";
  } else {
    return "Number not in range";
  }
}

console.log(numberRange(50));
console.log(numberRange(201));
console.log(numberRange(49));
console.log(numberRange(100));
console.log(numberRange(172));

File: number_range.py
import sys


def number_range(number):
    if isinstance(number, int):
        if int(number) >= 50 and int(number) <= 200:
            return "Number is valid"
        else:
            return "Number is invalid"
    else:
        return "Please enter a number"


#print(number_range(50))

print(number_range(sys.argv[1]))


